testkit/src/lib.rs

```diff
@@ -394,6 +394,7 @@ pub struct TestKitBuilder {
     us: TestNode,
     validators: Vec<TestNode>,
     services: Vec<Box<Service>>,
+    logger: bool,
```

Comment:
Personally, I prefer to name such boolean variables slightly different. For example, `enable_logger`. Not insisting though.